packages/newspringchurchapp/src/index.js

```diff
@@ -39,6 +40,7 @@ const AppNavigator = createStackNavigator(
     UserWebBrowser,
     Onboarding,
     Prayer,
+    PrayerList,
```

Comment:
Any reason why this couldn't be added to the PrayerNavigator (on the prayer tab) and not the global navigator. Seems like it would be more appropriate there?

Reply:
I believe this is the result of the nature of react-nav. I think to create the modal-like location, the component needs to exist on this global scale

Reply:
I added the `AddPrayerForm` to the prayer navigator and it's a modal

Reply:
it definitely needs to go in the prayer navigator, `onClose` takes me back to onboarding.

![Kapture 2019-05-01 at 10 17 05](https://user-images.githubusercontent.com/2659478/57021487-61e67500-6bfa-11e9-986d-c193f0752280.gif)

Reply:
That's odd. One of the guys from differential explain this to me and I don't recall what they said. @vinnyjth @conrad-vanl @burkeshartsis "HI!"

Reply:
Short answer: it depends. 

Long answer: In general if you can access a screen from anywhere in the app (not just a specific part of the app) you should put that screen inside the root navigator. 

However, if you are only using that screen from inside a subsection of the app, you are probably fine putting the route inside that subroute. That's because it's super easy to go "up the tree" for a route, but really hard to go "up and back down". I can explain that more if it doesn't make sense. 

Also, putting a route inside a TabNavigator can have consequences like needing to explicitly hide the tab. Looks like @redreceipt has already solved for that though.

Reply:
In this case, if prayer list is a modal that can be accessed from anywhere in the app, leave it there. If it's a modal that's only accessed by other screens in the prayer navigator, you can move it back down to the prayer navigator.

Reply:
@redreceipt I've actually seen that exact behavior on the discover tab in Core Apollos.

Reply:
Very odd.